pkg/virt-config/deprecation/feature-gates.go

```diff
@@ -28,9 +28,11 @@ import (
 type State string
 
 const (
-	GA           = "General Availability" // By default, GAed feature gates are considered enabled and no-op.
-	Deprecated   = "Deprecated"           // The feature is going to be discontinued next release
-	Discontinued = "Discontinued"
+	GA             = "General Availability" // By default, GAed feature gates are considered enabled and no-op.
```

Comment:
nit: I think we better move the comment above instead of in-lining.

Not a must though

Reply:
Done